migrations/20231102155822_add_sessions_tag.up.sql

```diff
@@ -0,0 +1,2 @@
+alter table if exists auth.sessions
```

Comment:
```suggestion
alter table if exists {{ index .Options "Namespace" }}.sessions
```

Reply:
Ahh yes!

Reply:
/nit can we bump the timestamp of the migraiotn closer towards merge? Aware that it's a bit dated bc there've a few chain PRs